web/src/features/charts/tooltips/CarbonChartTooltip.tsx

```diff
@@ -20,14 +20,19 @@ export default function CarbonChartTooltip({ zoneDetail }: InnerAreaGraphTooltip
   if (!zoneDetail) {
     return null;
   }
-  const { co2intensity, co2intensityProduction, stateDatetime, estimationMethod } =
-    zoneDetail;
+  const {
+    co2intensity,
+    co2intensityProduction,
+    stateDatetime,
+    estimationMethod,
+    estimatedPercentage,
+  } = zoneDetail;
   const intensity = getCarbonIntensity(
     isConsumption,
     co2intensity,
     co2intensityProduction
   );
-  const hasEstimationPill = estimationMethod != undefined;
+  const hasEstimationPill = estimationMethod != undefined || Boolean(estimatedPercentage);
```

Comment:
```suggestion
  const hasEstimationPill = Boolean(estimationMethod) || (estimatedPercentage > 0)
```
Might be good to use this to check if estimationMedtion is not null as well and that the % isn't 0

Reply:
But if the estimatedPercentage is 0 we don't want to show the pill?

Reply:
Yeah, Boolean(0) does the same thing anyway so shouldn't matter either way. 

The main change is Boolean(estimationMethod) so it checks for null and undefined